generators/heroku/generator.mjs

```diff
@@ -709,7 +709,48 @@ export default class HerokuGenerator extends BaseGenerator {
 
             this.logger.log(chalk.bold('\nConfiguring Heroku'));
             await execCmd(`heroku config:set ${configVars}--app ${this.herokuAppName}`);
-            await execCmd(`heroku buildpacks:add ${buildpack} --app ${this.herokuAppName}`);
+            const { stdout: data } = await execCmd(`heroku buildpacks:add ${buildpack} --app ${this.herokuAppName}`);
+            if (data) {
+              this.logger.info(data);
+              // remote:  !     The following add-ons were automatically provisioned: . These add-ons may incur additional cost,
+              // which is prorated to the second. Run `heroku addons` for more info.
+              if (data.includes('Run `heroku addons` for more info.')) {
+                this.spawnCommand('heroku addons');
```

Comment:
await it?

Reply:
Tested and `this.spawnCommand` throw errors (even with `await`), should use `await execCmd('heroku addons')` back.

<img width="657" alt="Screenshot 2023-04-23 at 19 44 07" src="https://user-images.githubusercontent.com/31362988/233837782-99ee633c-4fd9-4026-bb1e-fe9d7a13e5ad.png">